mr/build.gradle

```diff
@@ -41,21 +41,15 @@ project(':iceberg-mr') {
       exclude group: 'org.apache.avro', module: 'avro'
     }
 
-    compileOnly("${libs.hive2.exec.get().module}:${libs.hive2.exec.get().getVersion()}:core") {
```

Comment:
these changes don't seem to be related to removing Hive3?

Reply:
This PR also removes hive-runtime from `iceberg-mr`